docs/inside-arbitrum-nitro/inside-arbitrum-nitro.md

```diff
@@ -200,7 +200,7 @@ You might wonder why we need the rollup protocol. If everyone knows the results
 
 With those preliminaries behind us, let’s jump into the details of the rollup protocol.
 
-The parties who participate in the protocol are called _validators_. Anyone can be a validator. Some validators will choose to be stakers--they will place an ETH deposit which they’ll be able to recover if they’re not caught cheating. These roles are permissionless: anyone can be a validator or a staker.
+The parties who participate in the protocol are called _validators_. Some validators will choose to be stakers--they will place an ETH deposit which they’ll be able to recover if they’re not caught cheating. The protocol allows for these roles to be permissionless in principle. (Currently on Arbitrum One, validators/stakers are whitelisted; see ["mainnet beta status"](../mainnet-beta.md)).
```

Comment:
Good spot to describe `defensive` validators since that is what the majority of 3rd party validators will be running as.  Can also mention that anyone can run `watchtower` validator that logs an error when discrepancy is found.

Reply:
Also a good spot to point out that in the common case only one validator will be staked, because if one party is staked on the correct outcome, and nobody is staked on an incorrect outcome, there is no reason for others to stake.